src/transformers/__init__.py

```diff
@@ -706,6 +707,7 @@
     _import_structure["models.gpt2"].append("GPT2TokenizerFast")
     _import_structure["models.gpt_neox"].append("GPTNeoXTokenizerFast")
     _import_structure["models.gpt_neox_japanese"].append("GPTNeoXJapaneseTokenizer")
+    _import_structure["models.gptsan_japanese"].append("GPTSanJapaneseTokenizer")
```

Comment:
There is no need to put in this part of the init: the tokenizer has no dependency on tokenizers so it can be imported along with the config.